backend/app/dataspace_configuration.py

```diff
@@ -20,3 +20,17 @@ async def get_dataspace_configuration():
                 500, f"Error fetching dataspace configuration from {exc.request.url}"
             )
     return resp.json()
+
+
+async def get_oidc_provider_url() -> str:
+    dataspace_configuration = await get_dataspace_configuration()
+    oidc_provider_url = dataspace_configuration["authentication_providers"]["end_user"][
+        "base_url"
+    ]
```

Comment:
Is this the kind of formatting optimization thing you'd like to do? :smile: 

```suggestion
    auth_providers = dataspace_configuration["authentication_providers"]
    oidc_provider_url = auth_providers["end_user"]["base_url"]
```